### PR TITLE
[WFLY-16247]: Dump of Artemis cluster topology is hard to read.

### PR DESCRIPTION
**Pull request by ehsavoie**
* Reformatting the topology description before sending it back.

Jira: https://issues.redhat.com/browse/WFLY-16247

### CODE REVIEW

File: messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/ClusterConnectionControlHandler.java [hunk 1]
```diff
@@ -33,7 +32,8 @@
 import org.jboss.dmr.ModelNode;
 
 /**
- * Handler for runtime operations that interact with a ActiveMQ {@link org.apache.activemq.api.core.management.ClusterConnectionControl}.
+ * Handler for runtime operations that interact with a ActiveMQ
+ * {@link org.apache.activemq.api.core.management.ClusterConnectionControl}.
```
Comment (rhusar) [inline messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/ClusterConnectionControlHandler.java:36]: No need for a new line here...

File: messaging-activemq/subsystem/src/test/java/org/wildfly/extension/messaging/activemq/ClusterConnectionControlHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    public ClusterConnectionControlHandlerTest() {
+    }
+
+    /**
+     * Test of formatTopology method, of class ClusterConnectionControlHandler.
+     */
+    @Test
+    public void testFormatTopology() {
+        System.out.println("formatTopology");
```
Comment (rhusar) [inline messaging-activemq/subsystem/src/test/java/org/wildfly/extension/messaging/activemq/ClusterConnectionControlHandlerTest.java:35]: Unnecessary System.out..

**Comment by iweiss** [comment]
@ehsavoie I added a comment, did you check?

**Comment by iweiss** [comment]
LGTM

**Review by iweiss — APPROVED** [review]

**Review by rhusar — APPROVED** [review]

**Pull request merged**